template/module/__openerp__.py

```diff
@@ -0,0 +1,15 @@
+# -*- encoding: utf-8 -*-
+{
+    "name": "Module name",
+    "summary": "Module summary",
+    "version": "0.1.0",
+    "author": "Odoo Community Association (OCA)",
+    "category": "Module category",
+    "depends": [
+        "base",
+    ],
+    "demo": [
+    ],
+    "data": [
+    ],
```

Comment:
@bwrsandman I like your idea. Thus you mean the team which gather multiple repo? For exemple https://odoo-community.org/project/accounting-maintainers-9
Because we only direct to github page then.